docs/pages/es/concepts/architecture/broker.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Broker
+permalink: es/concepts/architecture/broker
+type: concepts
+redirect_from: architecture/broker
```

Comment:
@shubham-tyagi will you confirm?

Reply:
Ohh, yes it will @leecalcote. I completely forgot about this pull request. Thanks for the reminder. I guess the front matter variable redirect_from part needs to be updated. Am I in the right direction ?

Reply:
Spot on.

Reply:
Hey @chal-gro, can you set the front matter variable `redirect_from` to `es/architecture/broker`

Reply:
Hey @ramrodo, could you set the front_matter variable `redirect_from` to `es/architecture/broker`. So that the English page does not get redirected to the Spanish page.

Reply:
Hey @chal-gro, could you set the front_matter variable redirect_from to es/architecture/broker. So that the English page does not get redirected to the Spanish page.